src/AzureFunctions/Example/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
There is already .gitignore file in the root of repo. Could you use that existing file. You may add some entries if you need.

Reply:
Done